Description: Remove value assignments to environment variable LD_RUN_PATH
 Value assignment to environment variable LD_RUN_PATH leads to setted RPATH in
 compiled binary/shared library.
 .
 And then Lintian reports about error (E: binary-or-shlib-defines-rpath).
Author: Leonid Borisenko <leo.borisenko@gmail.com>
Forwarded: not-needed
Last-Update: 2013-04-24

--- a/plugins/rack/uwsgiplugin.py
+++ b/plugins/rack/uwsgiplugin.py
@@ -44,7 +44,6 @@
 
 if has_shared == 'yes':
     LDFLAGS.append('-L' + libpath )
-    os.environ['LD_RUN_PATH'] = libpath
     LIBS.append(os.popen(RUBYPATH + " -e \"require 'rbconfig';print '-l' + %s::CONFIG['RUBY_SO_NAME']\"" % rbconfig).read().rstrip())
 else:
     GCC_LIST.append("%s/%s" % (libpath, os.popen(RUBYPATH + " -e \"require 'rbconfig';print %s::CONFIG['LIBRUBY_A']\"" % rbconfig).read().rstrip()))
